docs/site/BelongsTo-relation.md

```diff
@@ -75,18 +98,44 @@ export interface OrderRelations {
 export type OrderWithRelations = Order & OrderRelations;
 ```
 
-The definition of the `belongsTo` relation is inferred by using the `@belongsTo`
-decorator. The decorator takes in a function resolving the target model class
-constructor and designates the relation type. It also calls `property()` to
-ensure that the decorated property is correctly defined.
+The above example can also be expanded with explicit relation and property
+definitions as follows,
+
+- The standard naming convention for the foreign key property in the source
+  model is `relation name` + `Id`. (e.g. Order.customerId).
```

Comment:
`relation name` + `Id` -> `source model name + Id`?

Reply:
@agnes512 do you mean `target model name + Id` ?

Reply:
ah, yes, I was thinking `Customer` is the source of this foreign key. But I think `target model` is more accurate according to the context.

Reply:
agreed